build.sbt

```diff
@@ -1094,33 +1108,14 @@ lazy val `project-manager` = (project in file("lib/scala/project-manager"))
       (`profiling-utils` / javaModuleName).value,
       (`ydoc-server` / javaModuleName).value
     ),
-    Test / modulePath := {
-      val updateReport = (Test / update).value
-      val requiredModIds =
-        GraalVM.modules ++ GraalVM.langsPkgs ++ logbackPkg ++ helidon ++ Seq(
-          "org.slf4j"        % "slf4j-api"                    % slf4jVersion,
-          "org.netbeans.api" % "org-netbeans-modules-sampler" % netbeansApiVersion
-        )
-      val requiredMods = JPMSUtils.filterModulesFromUpdate(
-        updateReport,
-        requiredModIds,
-        streams.value.log,
-        shouldContainAll = true
-      )
-      val runtimeMod =
-        (`runtime-fat-jar` / Compile / productDirectories).value.head
-      val ydocMod =
-        (`ydoc-server` / Compile / exportedProducts).value.head.data
-      val syntaxMod =
-        (`syntax-rust-definition` / Compile / exportedProducts).value.head.data
-      val profilingMod =
-        (`profiling-utils` / Compile / exportedProducts).value.head.data
-
-      requiredMods ++ Seq(
-        runtimeMod,
-        ydocMod,
-        syntaxMod,
-        profilingMod
+    Test / moduleDependencies := {
+      GraalVM.modules ++ GraalVM.langsPkgs ++ logbackPkg ++ helidon ++ Seq(
+        "org.slf4j"        % "slf4j-api"                    % slf4jVersion,
+        "org.netbeans.api" % "org-netbeans-modules-sampler" % netbeansApiVersion,
+        (`runtime-fat-jar` / projectID).value,
```

Comment:
This is how you specify that you want to put `runtime-fat-jar/target/classes` directory on module-path. I have found no way to infer this directly.